app/src/main/java/com/pvp/app/ui/screen/calendar/TaskCards.kt

```diff
@@ -87,25 +88,27 @@ fun TaskCard(
                     .padding(4.dp)
                     .fillMaxWidth()
             ) {
-                Checkbox(
-                    checked = checked,
-                    modifier = Modifier
-                        .size(36.dp)
-                        .align(CenterVertically),
-                    onCheckedChange = {
-                        model.update(
-                            { task ->
-                                Task.copy(
-                                    task,
-                                    isCompleted = it
-                                )
-                            },
-                            task
-                        )
+                if (task.date == LocalDate.now()) {
```

Comment:
I believe you made it so that no task that is in the past/future can be checked as completed. Some of our functionalities are specifically made to support past/future completions. We should discuss how we want to handle this, because if we drop support for past/future completions, I believe you will also have to adjust the surrounding code (workers, task service, maybe something else).